Tests/String+extensionTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  String+extensionTests.swift
+//  SwiftPrettyPrintTests
+//
+//  Created by Yusuke Hosonuma on 2020/02/20.
+//
+
+@testable import SwiftPrettyPrint
+import XCTest
+
+class String_extensionTests: XCTestCase {
+    override func setUp() {}
+
+    override func tearDown() {}
+
+    func test_indentTail() {
+        let text = """
+        Apple
+        Orange
+        Banana
+        """
+
+        XCTAssertEqual("Single line".indentTail(size: 4), "Single line")
+
+        XCTAssertEqual(text.indentTail(size: 2),
+                       """
+                       Apple
+                         Orange
+                         Banana
+                       """)
+
+        XCTAssertEqual(text.indentTail(size: 4),
+                       """
+                       Apple
+                           Orange
+                           Banana
+                       """)
+    }
+}
```

Comment:
I prefer the folloing form.
I think it make heredoc readable.
<img width="609" alt="Screen Shot 2020-02-21 at 9 02 09" src="https://user-images.githubusercontent.com/14083051/74991482-6336a600-5489-11ea-933d-224b496f1611.png">

Reply:
Good, I fixed it ( b1cbf73 )